### ISSUE TITLE
Bug: setting pugSingleFileComponentIndentation to true gets a conlicting rule on pug bracket interpolation

### ISSUE THREAD

**Issue by billyschonenberg** [prettier/plugin-pug#148]
## Info

| Tool     | Version       |
| -------- | ------------- |
| Plugin   | v1.10.0       |
| Prettier | v2.1.2        |
| Node     | v12.18.2    |
| OS       |  mac |

## Input
prettierrc.js
```js
module.exports = {
  pugSingleFileComponentIndentation: true,
}

```

file
```pug
<template lang="pug">
  div 
    span hello #[em world]
</template>
```

## Output or Error
alternating between:
```pug
<template lang="pug">
  div 
    span hello #[em world]
</template>
```

and 

```pug
<template lang="pug">
  div 
    span hello #[  em world]
</template>
```

## Expected Output
same as input.

**Comment by Shinigami92** [comment]
I'm sorry to inform you that I will probably not have time until the weekend to keep working on it

**Comment by billyschonenberg** [comment]
No problem, I just use the plugin as is with **pugSingleFileComponentIndentation** set to false.
Thanks for your work!

**Comment by Shinigami92** [comment]
This has now been fixed in release `1.10.1`
Thank you for using my plugin and for being part of the pug community 🤗
